Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/coil/CoilExtensions.kt

```diff
@@ -0,0 +1,21 @@
+package de.rki.coronawarnapp.util.coil
+
+import android.view.View
+import androidx.core.view.isInvisible
+import coil.request.ImageRequest
+
+fun ImageRequest.Builder.placeHolderView(
```

Comment:
I think the name here is confusing , it is not actually a place holder displayed by image itself, maybe 🤔 `loadingView` and we cloud pass the progressBar only?

Reply:
I've renamed it to `loadingView`.